vendor/k8s.io/kubernetes/openshift-kube-apiserver/admission/customresourcevalidation/oauth/validate_idp.go

```diff
@@ -72,10 +72,10 @@ func validateOAuthSpec(spec configv1.OAuthSpec) field.ErrorList {
 	}
 
 	// TODO move to ValidateTokenConfig
-	timeout := spec.TokenConfig.AccessTokenInactivityTimeoutSeconds
-	if timeout > 0 && timeout < MinimumInactivityTimeoutSeconds {
+	timeout := spec.TokenConfig.AccessTokenInactivityTimeout
+	if timeout != nil && timeout.Seconds() > 0 && timeout.Seconds() < MinimumInactivityTimeoutSeconds {
```

Comment:
can a timeout be negative?

Reply:
To allow the use case of not having a default timeout and still expire tokens if a timeout is set by oauth client.

https://coreos.slack.com/archives/GC3CZCQHM/p1594649609165700